build/images/base-windows/README.md

```diff
@@ -0,0 +1,22 @@
+# images/base-windows
+
+This Docker image includes download libraries for building windows image.
+
+If you need to build a new version of the image and push it to Dockerhub, you
+can run the following:
+
+```bash
+cd build/images/base-windows
+WIN_BUILD_TAG= echo $GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION| md5sum| head -c 10
+docker build -t antrea/base-windows:<WIN_BUILD_TAG> --network host .
+docker push antrea/base-windows:<WIN_BUILD_TAG>
+```
+
+The `docker push` command will fail if you do not have permission to push to the
+`antrea` Dockerhub repository.
+
+Here is the table of codegen images that have been uploaded:
+
+| Tag                              | Change                                  |
+| :-----------------------------   | --------------------------------------- |
+| fc89d4053c                       | The first image based on nanoserver-1809 |
```

Comment:
should be a43c77d755
but can we add a timestamp prefix so the tag has more information: which one is newer, when is it generated without having to query the table. for example: 20220510133001-fc89d4053c?